src/Model/Gallery.php

```diff
@@ -36,6 +36,11 @@ abstract class Gallery implements GalleryInterface
      */
     protected Collection $galleryItems;
 
+    public function __construct()
+    {
+        $this->galleryItems = new ArrayCollection();
```

Comment:
We should add some kind of changelog entry for this.

Reply:
Done